### PR TITLE
Fix user reports API docs

### PR DESCRIPTION
**Pull request by hanzei**
#### Summary

This is an example output of the API:
```json
[
    {
        "id": "a35iwstytb8m8xu3yejkm1fama",
        "create_at": 1726737170269,
        "update_at": 1726737215826,
        "delete_at": 0,
        "username": "dev.four",
        "auth_data": "",
        "auth_service": "ldap",
        "email": "success+devfour@simulator.amazonses.com",
        "nickname": "Dev4",
        "first_name": "Dev4",
        "last_name": "User",
        "position": "Staff Software Design Engineer",
        "roles": "system_user",
        "locale": "en",
        "timezone": {
            "automaticTimezone": "Europe/Berlin",
            "manualTimezone": "",
            "useAutomaticTimezone": "true"
        },
        "disable_welcome_email": false,
        "last_login": 1726737170291,
        "last_status_at": 1726737825387,
        "last_post_date": 1726737235922,
        "days_active": 1,
        "total_posts": 3
    },
]
```
The permission notices are also not correct. The user doesn't need to be an admin.
#### Ticket Link
None

#### Release Note
```release-note
NONE
```

### CODE REVIEW

**Review by devinbinnie — APPROVED** [review]
Thanks @hanzei :)

**Comment by mvitale1989** [comment]
The failing testcase is unrelated to this PR, and has been fixed on master branch (it would've succeeded after an update-branch).
I'll override the result of the E2E run to successful.

File: api/v4/source/reports.yaml [hunk 1]
```diff
@@ -22,10 +20,10 @@
             default: 'Username'
         - name: direction
           in: query
-          description: The direction in which to accept paging values from. Will return values ahead of the cursor if "up", and below the cursor if "down". Default is "down".
+          description: The direction in which to accept paging values from. Will return values ahead of the cursor if "prev", and below the cursor if "next". Default is "next".
```
Comment (cwarnermm) [inline api/v4/source/reports.yaml:23]:
```suggestion
          description: The direction to accept paging values from. Will return values ahead of the cursor if "prev", and below the cursor if "next". Default is "next".
```

**Review by ewwollesen — CHANGES_REQUESTED** [review]
I tested the permission change with not having to be a system_admin and was able to confirm that so it looks good. Added one minor change suggestion to the wording of one of the `description` fields.

File: api/v4/source/reports.yaml [hunk 2]
```diff
@@ -22,10 +22,10 @@
...
```
[the remark this answers is not in the document]
Comment (cwarnermm) [reply api/v4/source/reports.yaml]: Thanks, @ewwollesen! I've applied your suggestion.

**Review by ewwollesen — APPROVED** [review]
Looks good to me.

**Pull request merged**